spacecmd/spacecmd.spec

```diff
@@ -42,13 +47,17 @@ Name:           spacecmd
 Version:        4.0.12
 Release:        1%{?dist}
 Summary:        Command-line interface to Spacewalk and Red Hat Satellite servers
-License:        GPL-3.0-or-later
+%if %{_vendor} == "debbuild"
+Packager:       Uyuni packagers <uyuni-devel@lists.opensuse.org>
+Group:          admin
```

Comment:
According to [this doc](https://www.debian.org/doc/debian-policy/ch-archive.html#sections), I thin `admin`is the section that makes more sense.